### DISCUSSION TITLE
Erro ao conectar

### DISCUSSION THREAD

**Discussion by matheusvarela** [balta-io/2810#6] [Q&A]
Boa noite

Ao tentar fazer Produce ou consume está dando erro SSL handshake failed: Disconnected

Será que é o username e password?
Tem algum lugar que posso definir o usuário e senha?

**Comment by andrebaltieri** [comment]
@matheusvarela por favor, confere este tópico aqui:
https://github.com/balta-io/2805/discussions/5

**Reply by matheusvarela** [reply]
Boa noite

Como estava rodando local, "comentei" algumas linhas do arquivo csharp.config e foi, segue a alteração caso alguém precise

# Kafka
bootstrap.servers=localhost:29092
#security.protocol=SASL_SSL
#sasl.mechanisms=PLAIN
#sasl.username=HIDDEN_VALUE
#sasl.password=HIDDEN_VALUE
